lib/w3c-metadata.js

```diff
@@ -190,6 +194,17 @@ exports.previousVersion = function(url, publishDate, callback) {
         if (dates.length > 1) {
             callback(null, versions[dates[1]]);
             return;
+        } else if (dates[0]) {
+            var obsoletes = versions[dates[0]].obsoletes;
```

Comment:
does that mean there are cases where the list of versions is not complete yet obsoletes is set on the first version? isn't that a bug in the data rather than in the munger?